client/my-sites/backup/backup-contents-page/style.scss

```diff
@@ -117,14 +117,29 @@
 		}
 
 		.file-card {
+			background-color: var(--studio-gray-0);
```

Comment:
These styles are mostly for backward compatibility to ensure nothing is broken on the Jetpack Cloud's version.